WordPress/src/main/java/org/wordpress/android/viewmodel/StockMediaViewModel.java

```diff
@@ -0,0 +1,136 @@
+package org.wordpress.android.viewmodel;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+import android.arch.lifecycle.ViewModel;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.text.TextUtils;
+
+import org.greenrobot.eventbus.Subscribe;
+import org.greenrobot.eventbus.ThreadMode;
+import org.wordpress.android.fluxc.Dispatcher;
+import org.wordpress.android.fluxc.generated.StockMediaActionBuilder;
+import org.wordpress.android.fluxc.model.StockMediaModel;
+import org.wordpress.android.fluxc.store.StockMediaStore;
+import org.wordpress.android.util.AppLog;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+public class StockMediaViewModel extends ViewModel {
```

Comment:
We need the `start()` logic similar to the one in `PluginBrowserViewModel`. We won't need to do much in it, probably only need to do a new search if the view model wasn't started before and we have the search query. This should only happen if the activity is completely destroyed and re-created (not a config change).